decidim-core/app/helpers/decidim/scopes_helper.rb

```diff
@@ -19,5 +19,12 @@ def subscopes_for(scopable)
           Option.new(scope.id, translated_attribute(scope.name))
         end
     end
+
+    # Check whether the resource has a visible scope or not.
+    #
+    # Returns boolean.
+    def has_visible_scope?(resource)
```

Comment:
I'd pluralize `scope`, so `has_visible_scopes?`.

Also, I find it weird that you are using `present?` in one place but not on the other, can you use it in both? I prefer `if scope.present?` over `if scope`, but that's just my opinion